examples/gateway-httproute.yaml

```diff
@@ -110,7 +112,7 @@ spec:
     backendRefs:
     - name: httpbin
       port: 80
-      weight: 75
+      kind: Service
     - name: nginx
       port: 8080
-      weight: 25
+      kind: Service
```

Comment:
Can you help me to better understand this change?

Reply:
No reason, I just wanted to add the Kind, dunno why the weight is no more there 😕 I'll add it again.

Reply:
So for our implementation `Service` is practically all anyone would be using, but if you feel like it's much more illustrative to add it explicitly instead of relying on it being implicit, then I'm fine with that but yeah let's put the weights back as I feel these add some quick at-a-glance value to the samples.